src/components/EditProfile/EditProfile.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState, useEffect } from 'react'
+import axiosWithAuth from '../Auth/axiosWithAuth'
+import { Link } from 'react-router-dom'
+import './edit.css'
+
+const getStoreUrl = 'https://shopping-cart-eu3-staging.herokuapp.com/api/store/'
+const editStoreUrl = 'https://shopping-cart-eu3-staging.herokuapp.com/api/store'
```

Comment:
same URL?

Reply:
yeah one is  PUT request the other is a GET request

Reply:
Is it possible to just store URL and then use it for both? Would be good to avoid duplication if we can.

Reply:
If it's the same URL, do you need 2 variables?

Reply:
i can change it to one url